.circleci/config.yml

```diff
@@ -0,0 +1,154 @@
+          paths:
+            - .nyc_output/
+
+  test-node-10:
+    docker:
+      - image: node:10
+    steps:
+      - checkout
+      - run: npx nyc -s npm run test:node
+      - save_cache:
+          key: ipfs-{{ .BuildNum }}
+          paths:
+            - .nyc_output/
+
+  test-browser-chrome:
+    docker:
+      - image: hugomrdias/node-chrome:test]
+    steps:
+      - checkout
+      - run: AEGIR_BROWSERS=ChromeDocker npm run test:browser
+      - store_artifacts:
+          path: coverage/
+
+  test-browser-firefox:
+    docker:
+      - image: hugomrdias/node-firefox:test
+    steps:
+      - checkout
+      - run: AEGIR_BROWSERS=FirefoxHeadless npm run test:browser
+      - store_artifacts:
+          path: coverage/
+
+  cov:
+    docker:
+      - image: node:10-alpine
+    steps:
+      - checkout
+      - run: cp coverage/coverage-final.json .nyc_output/browser.json
+      - run: npx nyc report --reporter text-summary --reporter html
+      - store_artifacts:
+          path: coverage/
+
+
+  # github-auth:
+  #   docker:
+  #     - image: hugomrdias/node-alpine:test
+  #   steps:
+  #     - run: export GH_RELEASE_GITHUB_API_TOKEN=$GITHUB_TOKEN
+  #     - run: git remote add upstream https://$USER:$GITHUB_TOKEN@github.com/$USER/$CI_PROJECT_NAME.git
+  #     - run: git config --global user.email $EMAIL
+  #     - run: git config --global user.name $USER
+  #     - run: git checkout $CI_COMMIT_REF_NAME
+  #     - run: git fetch upstream
+  #     - run: git branch -u upstream/$CI_COMMIT_REF_NAME
+  #     - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+
+  pre-release:
+    docker:
+      - image: hugomrdias/node-alpine:test
+    steps:
+      - run: npm --no-git-tag-version version prerelease --preid=rc &>/dev/null
+      - run: npx conventional-changelog-cli -i CHANGELOG.md -r 0 -s -p angular
+      - run: git add CHANGELOG.md
+      - run: export version=`node -p "require('./package.json').version"`
+      - run: echo $version
+      - run: git commit -am "chore(pre-release) $version"
+
+  release:
+    docker:
+      - image: hugomrdias/node-alpine:test
+    steps:
+      - run: cp package.json _package.json
+      - run: bump=`npx conventional-recommended-bump -p angular`
+      - run: echo $bump
+      - run: npm --no-git-tag-version version $bump &>/dev/null
+      - run: npx conventional-changelog -i CHANGELOG.md -r 0 -s -p angular
+      - run: git add CHANGELOG.md
+      - run: version=`node -p "require('./package.json').version"`
+      - run: mv -f _package.json package.json
+      - run: git commit -am "docs(CHANGELOG) $version"
+      - run: npm version $bump -m "chore(release) v%s"
+      - run: git push --follow-tags upstream master
+      - run: npm publish
+      - run: npx gh-release -y
+
+  interop:
+    docker:
+      - image: node:10
+    steps:
+      - run: curl --request POST --form "token=$CI_JOB_TOKEN" --form ref=master https://gitlab.com/api/v4/projects/8952909/trigger/pipeline
```

Comment:
So this triggers a CI job on Gitlab for interop...it should use the commit that this job is building - looks like it just uses master?

Reply:
in my config this is meant to be a nightly triggers thats why it runs only master